linux_os/guide/system/auditing/auditd_freq/rule.yml

```diff
@@ -0,0 +1,30 @@
+documentation_complete: true
+
+title: 'Set number of records to cause an explicit flush to audit logs'
+
+description: |-
+    To configure Audit daemon to issue an explicit flush to disk command
+    after writing 50 records, set <tt>freq</tt> to <tt>50</tt>
+    in <tt>/etc/audit/auditd.conf</tt>.
+
+rationale: |-
+    If option <tt>freq</tt> isn't set to <tt>50</tt>, the flush to disk
+    may happen after higher number of record, increasing the danger
```

Comment:
probably records

Reply:
Fixed in 41a2debcab5fae88ccd15a62e87a5aef9381f3a2.